### ISSUE TITLE
Issues with webpack without babel-loader

### ISSUE THREAD

**Issue by primetoxinz** [chimera-js/vue-chimera#1]
When attempting to use vue-chimera with a webpack this error occurs.Not sure what could cause this
```
 error  in ./node_modules/vue-chimera/dist/vue-chimera.es.js

Module parse failed: Unexpected token (526:40)
You may need an appropriate loader to handle this file type.
|             resources = _objectWithoutProperties(_options$chimera, ["$options"]);
| 
|       _chimera = new VueChimera(this, { ...resources }, { ...config
|         ...$options
|       });

 @ ./src/plugins/chimera.js 2:0-37
 @ ./src/main.js
```

**Comment by SasanFarrokh** [comment]
Could you send your webpack configuration file? Thanks.

**Comment by SasanFarrokh** [comment]
Thanks a lot for submitting.
Fixed on new version [v2.1.1](https://github.com/chimera-js/vue-chimera/releases/tag/v2.1.1)
